pkg/metrics/common_test.go

```diff
@@ -0,0 +1,37 @@
+package mpcmetrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	prometheusTest "github.com/prometheus/client_golang/prometheus/testutil"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestRegisterCommonMetrics(t *testing.T) {
```

Comment:
@mshaposhnik I know that other tests in this repo were not written using Ginkgo - but can you make an effort to use it for new tests?

Also please try to use Gomega rather then testify for assertions